jobs/aws/eks-distro/main-1-18-postsubmits.yaml

```diff
@@ -50,6 +50,10 @@ postsubmits:
             - -c
             - date +%s > /status/pending
           periodSeconds: 10
+        resources:
+          limits:
```

Comment:
I think we may want to make this `requests` instead of `limits` to ensure when being scheduled it is guaranteed that much.  If we do that, we could probably leave the limit unbound and let it use as much as is available.

Reply:
Good point, I initially left as such because we had limits for others, but we can keep as requests too